crates/matrix-sdk-base/src/client.rs

```diff
@@ -1270,20 +1147,17 @@ impl BaseClient {
         changes: &StateChanges,
         room_info_notable_updates: BTreeMap<OwnedRoomId, RoomInfoNotableUpdateReasons>,
     ) {
-        if changes.account_data.contains_key(&GlobalAccountDataEventType::IgnoredUserList) {
-            if let Some(event) =
-                changes.account_data.get(&GlobalAccountDataEventType::IgnoredUserList)
-            {
-                match event.deserialize_as::<IgnoredUserListEvent>() {
-                    Ok(event) => {
-                        let user_ids: Vec<String> =
-                            event.content.ignored_users.keys().map(|id| id.to_string()).collect();
-
-                        self.ignore_user_list_changes.set(user_ids);
-                    }
-                    Err(error) => {
-                        warn!("Failed to deserialize ignored user list event: {error}")
-                    }
+        if let Some(event) = changes.account_data.get(&GlobalAccountDataEventType::IgnoredUserList)
```

Comment:
Asking to use the full processor name everywhere because I got confused here. Started looking for a get method in the processor. It's just simpler to not have that mental overload to being with.

Reply:
I'm not sure I follow, isn't `account_data` just a `BTreeMap`?

Reply:
Yeah, that's my point: we use the same `account_data` name for 2 different concepts in this file.

Reply:
Indeed, here it's not the processor per se, it's the `account_data` from `StateChanges`.